sdk/typescript/src/types/sui-bcs.ts

```diff
@@ -24,80 +24,6 @@ function registerUTF8String(bcs: BCS) {
   );
 }
 
-/**
```

Comment:
This is due to the merge with https://github.com/MystenLabs/sui/pull/9098 and https://github.com/MystenLabs/sui/pull/9101. Once these PRs lands, we can do a rebase